nifi-framework-bundle/nifi-framework/nifi-framework-core/src/main/java/org/apache/nifi/controller/FlowController.java

```diff
@@ -1067,6 +1067,16 @@ public void initializeFlow(final QueueProvider queueProvider) throws IOException
                 }
             }, 0L, 30L, TimeUnit.SECONDS);
 
+            // Parse flowcontroller sync delay/interval using FormatUtils
+            final String syncIntervalStr = nifiProperties.getProperty("nifi.flowcontroller.registry.sync.interval", "30 min");
+
+            // Convert to seconds
+            final long syncIntervalSec = FormatUtils.getTimeDuration(syncIntervalStr, TimeUnit.SECONDS);
+
+            // Log resolved values
+            LOG.info("Flow registry will sync every {}", syncIntervalStr);
```

Comment:
Minor adjustments on wording, and some of the comments and be removed:
```suggestion
            final String registrySyncInterval = nifiProperties.getProperty("nifi.flowcontroller.registry.sync.interval", "30 min");
            final long registrySyncIntervalSeconds = FormatUtils.getTimeDuration(registrySyncInterval, TimeUnit.SECONDS);

            LOG.info("Scheduled Flow Registry synchronization every {}", registrySyncInterval);
```